src/adapters/bidderFactory.js

```diff
@@ -250,6 +257,17 @@ export function newBidder(spec) {
     }
   });
 
+  function isUnknownBidder(unknownBidder, bidderCode) {
```

Comment:
I am not sure this works with aliases, nor that it can achieve the aim of https://github.com/prebid/Prebid.js/pull/8193 (batching requests from multiple bidders into one network call). Looking at how [aliases work](https://github.com/prebid/Prebid.js/blob/64bd041fa29e69605086dc094dfc525d9a4e0242/src/adapterManager.js#L479), I think they'd look as completely distinct adapters from here. The request batching seems like a bigger project unless I am misreading the code.

I think it should be a requirement (and also a good test case) that aliases set up both by the bid adapters and the publisher should *not* be flagged as "unknown" here, regardless of the `allowUnknown` settings.

Reply:
Hi @dgirardi ,
When a publisher sets an alias, as you mentioned `pbjs.aliasBidder('pubmatic', 'foobar')`.
Inputs to `isUnknownBidder` function are `bid.bidderCode as undefined` and `bidRequest.bidder as foobar` and even if adapter sends `bid.bidderCode as foobar` our condition inside `isUnknownBidder` function handles this scenario. 
Since it checks for following conditions in sequence

1. Valid unknownBidder
2. Valid requested bidder
3. if unknownBidder is not matching requested bidder
4. Then it checks for bidderSettings for unknownBidder

In case of aliases, the 3rd condition will not satisfy and function will return `false` and continue with bid from alias.

I hope it makes sense.

Reply:
Hi @dgirardi can you please share your thoughts on this.